docker/build.sh

```diff
@@ -74,7 +43,12 @@ else
 fi
 
 # Build the FlexFlow Enviroment docker image with input cuda version
-docker build --build-arg "FF_GPU_BACKEND=${FF_GPU_BACKEND}" --build-arg "cuda_version=${cuda_version}" -t "flexflow-environment-${FF_GPU_BACKEND}-${cuda_version}" -f docker/flexflow-environment/Dockerfile .
+# if FF_GPU_BACKEND, we don't tag the docker image with cuda version 
+if [[ "${FF_GPU_BACKEND}" == "hip_rocm" ]]; then
```

Comment:
here, to reduce code duplication, we can use the `if [[ "${FF_GPU_BACKEND}" == "hip_rocm" ]];` to set the cuda_version to variable to the empty string, and then use the same docker build command for both hip_rocm and CUDA